### DISCUSSION TITLE
Documentation of props from other components

### DISCUSSION THREAD

**Discussion by matsaal** [cdmoro/bootstrap-vue-3#2060] [General]
For components where a subset of the props are from another component (i.e. the number of components that include BLink) is it preferable for the component reference to have those props mixed in with the props that are strictly for the component itself, or for them to be added in an extension list (similarly to the BTable reference)?

**Comment by dwgray** [comment]
For cases like `BLink`, we've built a mechanism in the docs to pull in the property documentation into components that extend it. See the [Avatar](https://bootstrap-vue-next.github.io/bootstrap-vue-next/docs/components/avatar.html) and [Table](https://bootstrap-vue-next.github.io/bootstrap-vue-next/docs/components/table.html) docs for examples and #2093 for implementation.
